x-pack/plugins/infra/public/components/asset_details/links/link_to_apm_services.tsx

```diff
@@ -34,14 +34,16 @@ export const LinkToApmServices = ({ assetName, apmField }: LinkToApmServicesProp
   return (
     <RedirectAppLinks coreStart={services}>
       <EuiButtonEmpty
-        data-test-subj="infraAssetDetailsViewAPMServicesButton"
+        data-test-subj="infraAssetDetailsViewAPMShowAllServicesButton"
```

Comment:
I repurposed this for the "Show All" link.  This used to be the "APM Services" link at the top right of page.